android/build.gradle.kts

```diff
@@ -57,6 +59,14 @@ android {
             signingConfig = signingConfigs[BuildTypes.DEBUG]
         }
     }
+    flavorDimensions += "version"
+    productFlavors {
+        create("staging") {
```

Comment:
Could you please define the app name for [Staging](https://github.com/nimblehq/android-templates/blob/develop/template-compose/app/src/staging/res/values/strings.xml) and [Production](https://github.com/nimblehq/android-templates/blob/develop/template-compose/app/src/main/res/values/strings.xml)? We can skip defining app icons.

![image](https://github.com/nimblehq/kmm-templates/assets/16315358/54269632-9e78-4700-9c23-ad1d8142ef3c)